_im-plugin/performance.md

```diff
@@ -0,0 +1,134 @@
+sizes shows better indexing performance on ec2 instances.
+
+## Flush translog threshold
+
+Default value for `flush_threshold_size` is 512MB. This means that the translog is flushed when it reaches 512 MB. The weight of the indexing load determines the frequency of the translog. When you increase index.translog.flush_threshold_size, the node performs the translog operation less frequently. Because OpenSearch Service flushes are resource-intensive operations, reducing the frequency of translogs improves indexing performance. By increasing the flush threshold size, the OpenSearch Service cluster also creates fewer large segments (instead of multiple small segments). Large segments merge less often, and more threads are used for indexing instead of merging.
+
+For pure indexing workloads, consider increasing this value, e.g., 25% of the Java Heap size, to improve indexing performance.
+
+Note: An increase in index.translog.flush_threshold_size can also increase the time that it takes for a translog to complete. If a shard fails, then recovery takes more time because the translog is larger.
+
+Before increasing index.translog.flush_threshold_size, call the following API operation to get current flush operation statistics:
+
+```json
+curl -XPOST "os-endpoint/index-name/_stats/flush?pretty"
+```
+
+Replace the os-endpoint and index-name with your respective variables.
+
+In the output, note the number of flushes and the total time. The following example output shows that there are 124 flushes, which took 17,690 milliseconds:
+```json
+{
+     "flush": {
+          "total": 124,
+          "total_time_in_millis": 17690
+     }
+}
+```
+
+To increase the flush threshold size, call the following API operation:
+```json
+$ curl -XPUT "os-endpoint/index-name/_settings?pretty" -d "{"index":{"translog.flush_threshold_size" : "1024MB"}}"
+```
+
+In this example, the flush threshold size is set to 1024 MB, which is ideal for instances that have more than 32 GB of memory.
+
+Note: Choose the appropriate threshold size for your OpenSearch Service domain.
+
+Run the _stats API operation again to see whether the flush activity changed:
+```json
+$ curl _XGET "os-endpoint/index-name/_stats/flush?pretty"
+```
+
+Note: It's a best practice to increase the index.translog.flush_threshold_size only for the current index. After you confirm the outcome, apply the changes to the index template.
+
+## Index Refresh Interval
+
+By default, Opensearch periodically refreshes indices every second, but only on indices that have
+received one search request in the last 30 second1.
+
+When you increase the refresh interval, the data node makes fewer API calls. The refresh interval can be shorter or faster, depending on the length of the refresh interval. To prevent [429 errors](https://repost.aws/knowledge-center/opensearch-resolve-429-error), it's a best practice to increase the refresh interval.
```

Comment:
I'm not sure what's meant by this sentence "The refresh interval can be shorter or faster, depending on the length of the refresh interval." Does this mean we can change the refresh interval? If so, I don't think this sentence is needed.

Reply:
This got copy-pasted when I merged the doc from here per previous reviewer's suggestion: https://repost.aws/knowledge-center/opensearch-indexing-performance

I'll remove it as it doesn't make sense.